azure/resource_groups/app/template.json

```diff
@@ -32,6 +29,9 @@
       "type": "bool",
       "defaultValue": true
     },
+    "webAppServiceDockerImage": {
```

Comment:
Can this stay as `appServiceDockerImage`? It makes it clearer that it refers to the same thing as the other `appService*` things. Either that, or rename them all to `webAppService*`.